content/docs/hoon/reference/rune/lus.md

```diff
@@ -4,33 +4,50 @@ weight = 8
 template = "doc.html"
 +++
 
-A core is a cell of `[battery payload]`.  The battery is made of one or more arms, each of which is a computation on its parent core.
+A core is a cell of `[battery payload]`. The battery is made of one or more
+arms, each of which is a computation on its parent core.
 
-Arm runes are used to define arms in a core, and thus can only be used from within an expression that produces a multi-arm core.  That means arm runes cannot be used to mark the beginning of a first-class expression -- there is no such thing in Hoon as an arm without a core.
+Arm runes are used to define arms in a core, and thus can only be used from
+within an expression that produces a multi-arm core. That means arm runes cannot
+be used to mark the beginning of a first-class expression -- there is no such
+thing in Hoon as an arm without a core.
 
-There are various arm runes you can use to produce different kinds of arms.  Normal arms use `++`; arms defining a structure (or 'mold') use `+$`; and constructor arms use `+*`.
+There are various arm runes you can use to produce different kinds of arms.
+Normal arms use `++`; arms defining a structure (or 'mold') use `+$`; and
+constructor arms use `+*`.
 
-## Runes
-
-### `+|` "lusbar"
+## `+|` "lusbar"
 
 Chapter label.
 
-##### Syntax
+#### Syntax
 
-Regular: **1-fixed**.
+One argument, fixed.
 
-```hoon
+<table>
+<tr><th>Form</th><th>Syntax</th></tr>
+<tr>
+<td>Tall</td>
+<td>
+<pre>
 +|  %label
-```
+</pre>
+</td>
+</tr>
+<tr><td>Wide</td><td>None.</td></tr>
+<tr><td>Irregular</td><td>None.</td></tr>
+</table>
 
-##### Discussion
+#### Discussion
 
-The `+|` doesn't produce an arm.  It instead provides a label for the arms that follow it.  The arms of a core can be divided into **chapters** for 'organization'.  Chapter labels aren't part of the underlying noun of the core; they're stored as type system metadata only.
+The `+|` doesn't produce an arm. It instead provides a label for the arms that
+follow it. The arms of a core can be divided into **chapters** for
+'organization'. Chapter labels aren't part of the underlying noun of the core;
+they're stored as type system metadata only.
 
 See `tome` in the Hoon standard library.
 
-##### Examples
+#### Examples
```

Comment:
Let's put a pin to work on these.  These examples are distinctly unhelpful to someone deciding whether/how to use +|.

Reply:
lusbar is basically completely useless. You can't reference chapter labels short of manually processing the $type of a core. I can't think of a better example cos there's really isn't a practical example. I've added a couple of notes about how you wouldn't use this